app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee

```diff
@@ -374,6 +374,7 @@ ShortFormApplicationService = (
     else
       Service.preferences["#{prefType}_household_member"] = null
       Service.preferences["#{prefType}_proofOption"] = null
+      Service.preferences["#{prefType}_preference"] = null
```

Comment:
this is for resetting the value on check + uncheck. I don't think this will hurt live/work or rent burdened/assistive housing but we should probably check to make sure that the behavior for those doesn't change